modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlCommandHandler.java

```diff
@@ -88,8 +88,15 @@ public DdlCommandHandler(
         this.partitionIdleSafeTimePropagationPeriodMsSupplier = partitionIdleSafeTimePropagationPeriodMsSupplier;
     }
 
-    /** Handles ddl commands. */
-    public CompletableFuture<Boolean> handle(CatalogCommand cmd) {
+    /**
+     * Handles ddl commands.
+     *
+     * @param cmd Catalog command.
+     * @return Future representing pending completion of the operation. If the command execution resulted in a modification of the catalog,
+     *         the result will be the activation timestamp of the new catalog version, if the command did not result in a change of the
+     *         catalog, the result will be {@code null}.
+     */
+    public CompletableFuture<Long> handle(CatalogCommand cmd) {
```

Comment:
```suggestion
    public CompletableFuture<@Nullable Long> handle(CatalogCommand cmd) {
```

perhaps, it worth to add the annotation to every return type

Reply:
Fixed, thanks